src/Blueprint/Blueprint/Common/BlueprintConstants.cs

```diff
@@ -0,0 +1,53 @@
+﻿
+namespace Microsoft.Azure.Commands.Blueprint.Common
+{
+    public static class BlueprintConstants
+    {
+        public const string AzureBlueprintAppId = "f71766dc-90d9-4b7d-bd9d-4499c4331c3f";
+        public const string OwnerRoleDefinitionId = "/providers/Microsoft.Authorization/roleDefinitions/8e3af657-a8ff-443c-a75c-2fe8c4bcb635";
```

Comment:
@filizt what are these constants used for? Should they be used by the public?

Reply:
The AzureBlueprintAppId is used  to query Blueprint service's first party app object id and OwnerRoleDefinition is used as the definition id to assign owner role to blueprint first party app. They can both be public.

---
In reply to: [261404672](https://github.com/Azure/azure-powershell/pull/8537#discussion_r261404672) [](ancestors = 261404672)